src/mongo/util/net/httpclient.cpp

```diff
@@ -32,6 +32,7 @@
 #include "mongo/util/net/httpclient.h"
 
 #include "mongo/bson/util/builder.h"
+#include "mongo/base/parse_number.h"
```

Comment:
Please keep the includes in accordance with the [style guide](https://github.com/mongodb/mongo/wiki/Server-Code-Style#for-cpp-files).